NetCoreDemos/Views/Splitter/Overview.cshtml

```diff
@@ -2,44 +2,49 @@
     <div class="demo-container">
         @(Html.DevExtreme().Splitter()
             .ID("splitter")
-            .Items(items => {
+            .Items(items =>
+            {
                 items.Add()
                     .Resizable(true)
                     .Size("140px")
                     .MinSize("70px")
-                    .Template(new JS("function(data){return paneContentTemplate(data, 'Left Pane')}"));
+                    .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, itemElement, 'Left Pane')}"));
 
-                items.Add().Splitter(s => {
+                items.Add().Splitter(s =>
+                {
                     s.Orientation(Orientation.Vertical);
-                    s.Items(verticalItems => {
+                    s.Items(verticalItems =>
+                    {
                         verticalItems.Add()
                             .Resizable(true)
                             .Collapsible(true)
                             .MaxSize("75%")
-                            .Template(new JS("function(data){return paneContentTemplate(data, 'Central Pane')}"));
+                            .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, itemElement, 'Central Pane')}"));
 
                         verticalItems.Add()
                             .Collapsible(true)
-                            .Splitter(horizontalSplitter => {
+                            .Splitter(horizontalSplitter =>
+                            {
                                 horizontalSplitter.Orientation(Orientation.Horizontal);
-                                horizontalSplitter.Items(horizontalItems => {
+                                horizontalSplitter.Items(horizontalItems =>
+                                {
```

Comment:
looks like a bad formatting, lets revert this

Reply:
same above

Reply:
[c0c934e](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/c0c934ef05f6d28fb12291c26e9e5faad9f8aa6d)